src/bundle/Resources/public/scss/_tag.scss

```diff
@@ -0,0 +1,29 @@
+.ez-tag {
+    border-radius: calculateRem(4px);
+    background: $ez-color-base-pale;
+    padding: calculateRem(8px) calculateRem(48px) calculateRem(8px) calculateRem(8px);
+    display: inline-block;
+    position: relative;
+
+    &__content {
+        line-height: calculateRem(22px);
+    }
+
+    &__spinner {
+        height: calculateRem(22px);
+    }
+
+    &__remove-btn {
+        position: absolute;
+        top: calculateRem(7px);
+        right: calculateRem(6px);
+        cursor: pointer;
+        border: 0;
+        padding: 0;
+        outline: none;
+    }
+
+    &__remove-btn:focus {
+        outline: none;
```

Comment:
You should an alternative way to show the focus state. Isn't there any hover effect that could be applied as `focus` state as well?

Reply:
I based thin on how `.ez-field-edit--ezdate` works, so I'm not sure now, how we want our focuses to work. But I agree that it might be good idea to leave outline or replace it with something else. ping @sunpietro @dew326

Reply:
Please, add a focus state.